client/client_test.go

```diff
@@ -235,14 +239,19 @@ func TestClient_UpdateFeatures_Failure(t *testing.T) {
 		},`)
 
 	cfg := config.DefaultConfig()
-	cfg.Git.RepoPath = "/tmp"
+	dir, err := ioutil.TempDir("", "example")
+	if err != nil {
+		log.Fatal(err)
+	}
+	cfg.Git.RepoPath = dir
 	c, _ := New(cfg)
 	c.UpdateFeatures(badUpdate)
 	assert.EqualValues(t, models.EmptyFeatureMap(), c.FeatureMap(), "Assert bad payload returns empty feature map")
 }
 
 func TestWatch(t *testing.T) {
-	p := "/tmp/decider.json"
+	tmpfile, err := ioutil.TempFile("", "example")
```

Comment:
Let's name this `decider.json` to be in line with what we are expecting. I know that the library assumes "" to be the default temporary directory, but maybe we can be more explicit with: `os.TempDir()`.

It seems like it's up to us to clean up this file as well.